lib/main.ts

```diff
@@ -60,14 +63,29 @@ class AchYouDataProviderEngine
     return true;
   }
 
+  /**
+   * @param selector string userId of new user
+   * @param data date and any comments associated with the sneeze
+   */
+  // Note: this method gets layered over so that you can run `this.set` in the data provider engine,
+  // and it will notify update afterward.
+  async setUser(selector: string, data: User) {
+    if (selector === 'NEWUSER') {
+      logger.log('About to push new user');
+      logger.log(`data.name: ${(data as User).name}`);
+      this.users.push(data as User);
+    }
+
+    return true;
```

Comment:
Returning `true` sends a notification to subscribers that they need to update their data. You can make this more efficient by returning `true` only when you make edits like in the `if` block above and then returning `false` otherwise (meaning there were no changes and subscribers do not need to update).